### DISCUSSION TITLE
Creating a normal index DX

### DISCUSSION THREAD

**Discussion by kjoedion** [sequelize/sequelize#15310] [Help & Questions] [answered]
How do I create a single non-unique index on a single column?

Is this the recommended approach?:

```js
db.define(
  "User",
  {
    name: DataTypes.STRING,
    email: DataTypes.STRING,
    password: DataTypes.STRING,
  },
  {
    indexes: [
      {
        fields: ["name"],
      },
    ],
  }
);
```

It would be awesome if I could just do this instead (the same way I can with `unique`):

```js
db.define("User", {
  name: { type: DataTypes.STRING, index: true },
  email: DataTypes.STRING,
  password: DataTypes.STRING,
});
```

Is the first example the only way?

**Comment by ephys** [comment, marked as answer]
The first example is the only way

I don't know if we're ever going to add the second form for non-uniques. It's not used enough to justify having multiple ways to define them

**Reply by ephys** [reply]
To follow-up on this: Since we're including all Sequelize-TypeScript operators, including [`@Index`](https://github.com/sequelize/sequelize-typescript/#index), it may make sense for symmetry to accept this option on individual columns too

**Reply by ephys** [reply]
Implemented in this: https://github.com/sequelize/sequelize/pull/15431
We already had it but it only worked for JSONB columns. I've expended it to all columns during this refactor
